### PR TITLE
include Tk as dependency in Python 2.7.9 easyconfigs

### PR DESCRIPTION
**Pull request by boegel**
@JensTimmerman: please review

### CODE REVIEW

File: easybuild/easyconfigs/t/Tk/Tk-8.6.4-GCC-4.9.2-no-X11.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+version = '8.6.4'
+versionsuffix = '-no-X11'
+
+homepage = 'http://www.tcl.tk/'
+description = """Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for building
+ a graphical user interface (GUI) in many different programming languages."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+
+source_urls = ["http://prdownloads.sourceforge.net/tcl"]
```
Comment (JensTimmerman) [inline easybuild/easyconfigs/t/Tk/Tk-8.6.4-GCC-4.9.2-no-X11.eb]: Why not use the `SOURCEFORGE` constant here?

Comment (boegel) [inline easybuild/easyconfigs/t/Tk/Tk-8.6.4-GCC-4.9.2-no-X11.eb]: URL doesn't match the software name (Tk vs `tcl`)

**Comment by JensTimmerman** [comment]
I'm not sure you can just change these existing easyconfigs to add a new tk as a dependency, this breaks easyconfigs who now depend on python and a different version of tk..

**Comment by boegel** [comment]
Hmm, good point...

**Comment by boegel** [comment]
@JensTimmerman: including Tk 8.6.3 as a dep works, i.e. no conflicts are introduced in existing easyconfigs

I consider this a bug fix, so feel it's OK if this has effect on easyconfigs that haven't been merged centrally yet.

Thoughts?

**Comment by JensTimmerman** [comment]
Sure, it works for our repository, but what about people's private EC's?

**Comment by JensTimmerman** [comment]
Well, other people might not agree, it could break the build of private easyconfigs, I'm not sure what our policy there was, weren't we going to try not to break things?

**Comment by boegel** [comment]
@JensTimmerman: we maintain backward compatibility, except when bug fixes are required, then (minor) breaking changes can be considered

I consider the missing Tk dep a bug, since things don't work because of it

People should move on to Python 2.7.10 anyway ;)

**Comment by boegel** [comment]
@JensTimmerman: this will be clearly mentioned in the release notes, and people can still either use an older easyconfig that does not include the Tk dep, or use `--filter-deps=Tk`, if they feel they need to

**Comment by boegel** [comment]
Also, I just realised there's no need for this whole discussion anyway: `Tcl` v8.6.3 is already an indirect dependency to Python, via SQLite, and since the `Tcl` version also dictates the `Tk` version, there's no room in practice for any other `Tk` versions.

**Comment by JensTimmerman** [comment]
OK, good to go

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 14 out of 14 (14 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/42b2371430a154b35c3e for a full test report.

**Comment by boegel** [comment]
Going in, thanks for the critical review @JensTimmerman!

**Pull request merged**